homeassistant/components/clicksend_tts/notify.py

```diff
@@ -23,20 +24,27 @@
 
 CONF_LANGUAGE = "language"
 CONF_VOICE = "voice"
-CONF_CALLER = "caller"
 
+MALE_VOICE = "male"
+FEMALE_VOICE = "female"
+
+DEFAULT_NAME = "clicksend_tts"
 DEFAULT_LANGUAGE = "en-us"
-DEFAULT_VOICE = "female"
+DEFAULT_VOICE = FEMALE_VOICE
 TIMEOUT = 5
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
I think the config validation changes are ok as we're not introducing new options, but setting defaults and improving validation and removing incorrect options.